app/jobs/upvs/fetch_digital_services_and_forms_list_job.rb

```diff
@@ -0,0 +1,52 @@
+require 'csv'
+require 'harvester_utils/downloader'
+
+class Upvs::FetchDigitalServicesAndFormsListJob < ApplicationJob
+  queue_as :upvs
+
+  def perform(url, downloader: HarvesterUtils::Downloader)
+    csv_file = downloader.download_file(url)
+    csv_options = { col_sep: File.open(csv_file) { |f| f.readline }.include?(';') ? ';' : ',', headers: true }
+
+    TemporaryDigitalServiceAndForm.transaction do
+      TemporaryDigitalServiceAndForm.create_table!
+
+      each_row_as_attributes(csv_file, csv_options) do |attributes|
+        TemporaryDigitalServiceAndForm.find_or_initialize_by(uri: attributes[:uri]).update!(attributes)
+      end
+
+      TemporaryDigitalServiceAndForm.truncate_source_table!
+      TemporaryDigitalServiceAndForm.insert_to_source_table!
+    end
+  end
+
+  class TemporaryDigitalServiceAndForm < TemporaryRecord
+    def self.source
+      Upvs::DigitalServiceAndForm
+    end
+  end
+
+  private
+
+  def each_row_as_attributes(csv_file, csv_options)
+    CSV.foreach(csv_file, csv_options) do |row|
+      row = row.to_h.transform_keys { |k| k.to_s }
+
+      yield(
+        id_service_instance: row.fetch('IdServiceInstance'),
```

Comment:
tu by som siel s `_id` na koniec a slovo `instance` vypustil

```suggestion
        service_id: row.fetch('IdServiceInstance'),
```

Reply:
tu som sa pomylil, spravil by som to tak, ze by som slovo `service` vypustil

```suggestion
        instance_id: row.fetch('IdServiceInstance'),
```

Reply:
ono, kedze sa ten model vola `DigitalServiceAndForm` mozno by nebolo od veci prefixovat atributy `service_` alebo `form_` nech je jasne co je co, ale toto asi nehrotme...